README.md

```diff
@@ -1,214 +1,174 @@
-applications Appium has grown to a full-featured platform that provides [WebDriver](https://www.w3.org/TR/webdriver/)-based automation possibilities for the whole set of different mobile and desktop platforms.
-See the full list of
-[officially-supported](https://appium.io/docs/en/latest/ecosystem/#drivers) and
-[third-party](https://appium.io/docs/en/latest/ecosystem/#other-drivers) drivers for more details.
+<p align="center"><b>
+   <a href="https://appium.io">Documentation</a> |
+   <a href="https://appium.io/docs/en/latest/intro/">Get Started</a> |
+   <a href="https://appium.io/docs/en/latest/ecosystem/">Ecosystem</a> |
+   <a href="https://github.com/appium/appium/blob/master/packages/appium/CHANGELOG.md">Changelog</a> |
+   <a href="https://github.com/appium/appium/blob/master/ROADMAP.md">Roadmap</a> |
+   <a href="https://discuss.appium.io">Discussion Forum</a>
+</b></p>
 
-:bangbang: Appium core team does not maintain Appium 1.x anymore since the 1st of January 2022. All recent versions of officially supported platform drivers are not compatible to Appium 1.x, and require Appium 2 to run. [Please read the migration guide from 1.x to 2.0](https://appium.io/docs/en/latest/guides/migrating-1-to-2/) to manage the Appium server.
+***
 
-:bangbang: If you still use the deprecated Appium Desktop containing the built-in server then you won't be able to upgrade to Appium 2. Consider switching to [Appium Inspector](https://github.com/appium/appium-inspector) instead.
+Appium is an open-source automation framework that provides
+[WebDriver](https://www.w3.org/TR/webdriver/)-based automation possibilities for a wide range of
+different mobile, desktop and IoT platforms. Appium is modular and extensible, and supports multiple
+programming languages, which means there is an entire ecosystem of related software:
+* [__Drivers__](#drivers) add support for automating specific platforms
+* [__Clients__](#clients) allow writing Appium tests in your programming language of choice
+* [__Plugins__](#plugins) allow to further extend Appium functionality
 
-### Documentation
+## Migrating From Appium 1 to Appium 2
 
-The current Appium documentation is available at [__https://appium.io__](https://appium.io).
+As of January 1st, 2022, the Appium team no longer maintains or supports Appium 1. All officially
+supported platform drivers are only compatible with Appium 2.
 
-### Requirements
+__[Please read the Migration Guide](https://appium.io/docs/en/latest/guides/migrating-1-to-2/) if you
+are still using Appium 1.__
 
-- macOS, Linux or Windows operating system
-- Node.js 16+. Any relevant [LTS release](https://nodejs.dev/en/about/releases/) should work just fine.
-- NPM (Node Package Manager) 8+
+If you use Appium Desktop or Appium Server GUI, you will not be able to upgrade to Appium 2, as
+both of these tools have been deprecated. Please use [Appium Inspector](https://github.com/appium/appium-inspector)
+in combination with a standalone Appium 2 server.
```

Comment:
You can use the new markdown alerts.
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts
In this case I whould use the WARNING alert i.e.